test/jdk/java/lang/Math/ExpCornerCaseTests.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2011, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Before pushing, [lease update the copyright year to "2011, 2020" (since this file was based on one from 2020).